.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,110 @@
+## This workflow is triggered on pushes to the master branch and on pull requests.
+name: Terminus 3.x
+on:
+  push:
+    ignore: [ 'master' ]
```

Comment:
Should this be "ignore"? The comment says it is triggered on pushes to master, so I expect this should be "branches".

Reply:
Build and test runs on PR and pushes to every non-master branch. Merged runs when something is merged into the 3.x branch and release runs when a release is created.

This is telling it to ignore pushes to master because we don't care about those.